.github/workflows/test.yml

```
@@ -37,7 +37,7 @@ jobs:
         runs-on:
           - windows-latest
           - ubuntu-latest
-          - macos-13
+          - macos-latest
```

```suggestion
          - macos-14
```
I think this might fix the CI errors